sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+from ...management._xml_workaround_policy import ServiceBusXMLWorkaroundPolicy
+from ...management._handle_response_error import _handle_response_error
+from ...management._model_workaround import avoid_timedelta_overflow
+
+
+if TYPE_CHECKING:
+    from azure.core.credentials_async import AsyncTokenCredential  # pylint:disable=ungrouped-imports
+
+
+class ServiceBusManagementClient:  #pylint:disable=too-many-public-methods
+    """Use this client to create, update, list, and delete resources of a ServiceBus namespace.
+
+    :param str fully_qualified_namespace: The fully qualified host name for the Service Bus namespace.
+    :param credential: To authenticate to manage the entities of the ServiceBus namespace.
+    :type credential: Union[TokenCredential, ServiceBusSharedKeyCredential]
+    """
+
+    def __init__(self, fully_qualified_namespace, credential, **kwargs):
+        # type: (str, Union[AsyncTokenCredential, ServiceBusSharedKeyCredential], Dict[str, Any]) -> None
+
+        self.fully_qualified_namespace = fully_qualified_namespace
+        self._credential = credential
+        self._endpoint = "https://" + fully_qualified_namespace
+        self._config = ServiceBusManagementClientConfiguration(self._endpoint, **kwargs)
+        self._pipeline = self._build_pipeline()
+        self._impl = ServiceBusManagementClientImpl(endpoint=fully_qualified_namespace, pipeline=self._pipeline)
+
+    async def __aenter__(self) -> "ServiceBusManagementClient":
+        await self._impl.__aenter__()
+        return self
+
+    async def __aexit__(self, *exc_details) -> None:
+        await self._impl.__aexit__(*exc_details)
+
+    def _build_pipeline(self, **kwargs):  # pylint: disable=no-self-use
+        transport = kwargs.get('transport')
+        policies = kwargs.get('policies')
+        credential_policy = \
+            AsyncServiceBusSharedKeyCredentialPolicy(self._endpoint, self._credential, "Authorization") \
+            if isinstance(self._credential, ServiceBusSharedKeyCredential) \
+            else AsyncBearerTokenCredentialPolicy(self._credential, JWT_TOKEN_SCOPE)
+        if policies is None:  # [] is a valid policy list
+            policies = [
+                RequestIdPolicy(**kwargs),
+                self._config.headers_policy,
+                self._config.user_agent_policy,
+                self._config.proxy_policy,
+                ContentDecodePolicy(**kwargs),
+                ServiceBusXMLWorkaroundPolicy(),
+                self._config.redirect_policy,
+                self._config.retry_policy,
+                credential_policy,
+                self._config.logging_policy,
+                DistributedTracingPolicy(**kwargs),
+                HttpLoggingPolicy(**kwargs),
+            ]
+        if not transport:
+            transport = AioHttpTransport(**kwargs)
+        return AsyncPipeline(transport, policies)
+
+    @classmethod
+    def from_connection_string(cls, conn_str, **kwargs):
```

Comment:
nitpick: public methods should be put after private methods

Reply:
Updated